frontend/src/context/UserInputContext.tsx

```diff
@@ -0,0 +1,32 @@
+import { BigNumber } from "ethers"
```

Comment:
I know we use this in multiple places and it would be easiest to put this in a context, but in this case, it is better to make use this as a component and store whatever state you need there. And consequently, pass down extra props to other components as needed.

This keeps the components dumb and simple and easily interchangeable. With a context, these things become a little too coupled.

Reply:
Moved the state management to `GeyserStakeView` component